lib/localdatabase.js

```diff
@@ -0,0 +1,96 @@
+/*!
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const Dexie = require("dexie"),
+      indexedDB = require("fake-indexeddb"),
+      IDBKeyRange = require("fake-indexeddb/lib/FDBKeyRange");
```

Comment:
These are just for tests, right? Or would you need them if you were running this via node?

I wonder if there's a better way to inject fake IndexedDB APIs, but this is probably fine...

Reply:
It's needed to run in node, regardless of testing.

There's not a better way that I've found, and this kind of pattern is pretty common in modules that have to do some juggling between node and web.